keras/applications/inception_v3.py

```diff
@@ -27,7 +27,7 @@
 from ..utils.layer_utils import convert_all_kernels_in_model
 from ..utils.data_utils import get_file
 from .. import backend as K
-from .imagenet_utils import decode_predictions, _obtain_input_shape
```

Comment:
This should be kept (for namespace consistency: you want to be able to import `inception_v3.decode_predictions`).